ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp

```diff
@@ -106,7 +107,7 @@ ParallelConfig determine_parallel_config(
         if(shard_layout == TensorMemoryLayout::HEIGHT_SHARDED) {
             return find_closest_largest_divisor(conv_out_2d_matrix_height_ntiles, max_num_cores);
         } else if(shard_layout == TensorMemoryLayout::BLOCK_SHARDED) {
-            return find_closest_largest_divisor_with_num_padding(conv_out_2d_matrix_height_ntiles, device_grid_size[0]);
+            return find_closest_largest_divisor_with_num_padding(conv_out_2d_matrix_height_ntiles, device_grid_size[1]);
```

Comment:
Why the change?

Reply:
This is outdated, but a similar change has been made in the updated version.  This was a bug in `determine_parallel_config`.  The divisor needs to be either index 1 or 0 depending on whether the shard orientation is column or row major.